source/04-utility/_slide.es6.js

```diff
@@ -9,108 +9,112 @@
  * Slides target element up and out view.
```

Comment:
I wonder if we still need this - feels like we do these kind of transitions mostly with css these days. @kmonahan any thoughts?

Reply:
Literally just used this on a Drupal project last week, so yes, I think it is still useful and should stay.